sqlglot/expressions.py

```diff
@@ -330,7 +334,17 @@ def __hash__(self):
 
 
 class Insert(Expression):
-    arg_types = {"this": True, "expression": True, "overwrite": False, "exists": False}
+    arg_types = {
+        "this": True,
+        "expression": True,
+        "overwrite": False,
+        "exists": False,
+        "partition": False,
+    }
+
+
+class Partition(Expression):
+    arg_types = {"values": True}
```

Comment:
ok, if it's only values, then let's just not override arg_types, just do pass here and use this keyword